interface/main/messages/js/reminder_appts.js

```diff
@@ -0,0 +1,524 @@
+/**
+ * /interface/main/messages/js/reminder_appts.js
+ *
+ * Copyright (C) 2017 MedEx <support@MedExBank.com>
+ *
+ * LICENSE: This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * @package OpenEMR
+ * @author MedEx <support@MedExBank.com>
+ * @link http://www.open-emr.org
+ */
```

Comment:
Lets shorten this license/copyright/author header to just:
``````
/**
 * Message center javascript.
 *
 * @package OpenEMR
 * @link    http://www.open-emr.org
 * @author  MedEx <support@MedExBank.com>
 * @copyright Copyright (c) 2017 MedEx <support@MedExBank.com>
 * @license https://www.gnu.org/licenses/agpl-3.0.en.html GNU Affero General Public License 3
 */